From 4a97d099013714c3b529683946ed2ab6e7196f44 Mon Sep 17 00:00:00 2001
From: Cody Lacey <clacey@ti.com>
Date: Fri, 9 Aug 2013 13:40:35 -0500
Subject: [PATCH 23/27] drm/tilcdc increase allowable supported resolution

1680x1050 appears to also be within the bandwidth capabilities
of the device and memory infrastructure.
Created by: Darren Etheridge <detheridge@ti.com>
---
 drivers/gpu/drm/tilcdc/tilcdc_drv.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/tilcdc/tilcdc_drv.h b/drivers/gpu/drm/tilcdc/tilcdc_drv.h
index 254bba60..b4562d5 100644
--- a/drivers/gpu/drm/tilcdc/tilcdc_drv.h
+++ b/drivers/gpu/drm/tilcdc/tilcdc_drv.h
@@ -44,7 +44,7 @@
  * be supportable 
  * Note: 1920x1080x25=49766400 < 1280x1024x60=78643200
  */
-#define TILCDC_DEFAULT_MAX_BANDWIDTH  (1280*1024*60)
+#define TILCDC_DEFAULT_MAX_BANDWIDTH  (1680*1050*60)
 
 struct tilcdc_drm_private {
 	void __iomem *mmio;
-- 
1.7.9.5

